app/views/layouts/_header.html.erb

```diff
@@ -4,9 +4,9 @@
       <%= link_to 'News', root_path, class: 'btn btn-warning' %>
       <%= link_to 'About Us', abouts_path, class: 'btn btn-warning' %>
       <%= link_to 'Contacts', contacts_path, class: 'btn btn-warning' %>
-      <% if user_signed_in? %>
+      <% if user_signed_in? && current_user.account %>
```

Comment:
How make LogOut when user_signed_in but without account?

Reply:
How better to do, like now I update or like this using account_helper to avoiding overused if statements in html?
```module AccountsHelper
    def link(account)
    account ? account_path(account) : ''
  end
end
```
*html*
   <%= link_to 'Account', link(current_user.account), class: 'btn btn-warning right' %>